pkg/promclient/promclient.go

```diff
@@ -28,6 +29,12 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var FlagsVersion *version.Version
```

Comment:
Missing comment here on an exported variable. Plus, you can remove `init` and write here:

`var FlagsVersion *version.Version = version.Must(version.NewVersion("2.2.0"))`